plugins/vocabbuilder.koplugin/main.lua

```diff
@@ -2058,14 +2058,27 @@ function VocabBuilder:onDictButtonsReady(dict_popup, buttons)
         text = _("Add to vocabulary builder"),
         font_bold = false,
         callback = function()
-            local book_title = (dict_popup.ui.doc_props and dict_popup.ui.doc_props.display_title) or _("Dictionary lookup")
-            dict_popup.ui:handleEvent(Event:new("WordLookedUp", dict_popup.lookupword, book_title, true)) -- is_manual: true
             local button = dict_popup.button_table.button_by_id["vocabulary"]
-            if button then
-                button:disable()
+            if not button then return end
+            if button.text == _("Add to vocabulary builder") then
```

Comment:
That seems a tad unexpected as a condition compared to some other mode or property?

Reply:
I thought it was the obvious way without adding another property. Should I add an ad-hoc property to the button like `is_remove` instead?

Reply:
Done.